dexmaker/src/main/java/com/android/dx/stock/ProxyBuilder.java

```diff
@@ -949,6 +944,8 @@ public int hashCode() {
     private static class ProxiedClass<U> {
         final Class<U> clazz;
 
+        final Set<?> interfaces;
```

Comment:
Does interface order matter for default inheritance with default methods?

Reply:
Originally the order of interfaces is not considered.

https://github.com/linkedin/dexmaker/pull/125/files/ab6d870302154e502a693337944d550a31cbb2f6#diff-50d57f6ae81387a5d38ed25e3e7b0dd9L278

I'm not sure if we should consider that order in this PR.

Reply:
Let's do it now to avoid another bug.

Reply:
Changed ProxiedClass to consider interface order